packages/era/tsconfig.prod.esm.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../config/tsconfig.prod.esm.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist/esm",
+    "composite": true
+  },
+  "include": ["src/**/*.ts"],
+  "references": [{ "path": "../rlp/tsconfig.prod.esm.json" }]
```

Comment:
I think you likely need to add block and blockchain here so we don't end up with weird compilation errors down the line.